frontend/layouts/project-page/developers/component.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import Image from 'next/image';
+
+import LayoutContainer from 'components/layout-container';
+import { ProjectDevelopersProps } from 'layouts/project-page/developers/types';
+
+export const ProjectDevelopers: React.FC<ProjectDevelopersProps> = ({
+  project,
+}: ProjectDevelopersProps) => {
+  const NUMBER_DEVELOPERS = project.project_developer.length || 1;
+
+  const {
+    project_developer: { about, name, picture, project_developer_type },
+  } = project;
+  return (
+    <section className="bg-background-middle py-18">
+      <LayoutContainer className="flex space-x-28">
+        <div className="flex flex-col pl-16 space-y-1">
+          <h2 className="font-serif text-3xl text-black ">
+            <FormattedMessage defaultMessage="Project Developers" id="+K9fF0" />
+          </h2>
+          <p className="text-gray-800">
+            <FormattedMessage
+              defaultMessage="This project has {numDevelopers} project developer{noun}"
+              id="CcFPkO"
+              values={{ numDevelopers: NUMBER_DEVELOPERS, noun: NUMBER_DEVELOPERS > 1 ? 's' : '' }}
+            />
+          </p>
+        </div>
+        <div className="w-2/3 p-6 space-y-8 font-sans bg-white border rounded-2xl border-beige">
```

Comment:
This should be a list of the `project_developer` and `involved_project_developers`. 

We have a [profile card](https://github.com/Vizzuality/heco-invest/tree/develop/frontend/containers/profile-card) we can use (plus it'd like to the respective project developer profiles).